workspaces/quay/.changeset/loud-radios-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-quay-backend': minor
```

Comment:
Does this really need to be `minor` feels like `patch`?

Reply:
Hi @awanlin , changed to a `patch`. Thanks!